src/Compiler/Interactive/FSIstrings.txt

```diff
@@ -58,3 +58,4 @@ fsiOperationCouldNotBeCompleted,"Operation could not be completed due to earlier
 fsiOperationFailed,"Operation failed. The error text has been printed in the error stream. To return the corresponding FSharpDiagnostic use the EvalInteractionNonThrowing, EvalScriptNonThrowing or EvalExpressionNonThrowing"
 fsiMultiAssemblyEmitOption,"Emit multiple assemblies (on by default)"
 2304,fsiEntryPointWontBeInvoked,"Functions with [<EntryPoint>] are not invoked in FSI. '%s' was not invoked. Execute '%s <args>' in order to invoke '%s' with the appropriate string array of command line arguments."
+fsiDetailedHelpLink,"See https://learn.microsoft.com/en-us/dotnet/fsharp/language-reference/fsharp-interactive-options for more details."
```

Comment:
Link shouldn't contain language, it can just be removed, then it will redirect you to the language your browser sends as accepted.